### PR TITLE
Update configuration for docker

### LINKED ISSUE #11: Update configuration json to reflect Configuration Struct
Referenced in Issue #10

### PR DESCRIPTION
**Pull request by spencerbull**
Signed-off-by: Spencer Bull <spencer_bull@dell.com>

#10 Resolved
#11 Resolved
#13 Revision

### CODE REVIEW

File: res/configuration-docker.json [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
+	"ServerTimeout": 5000,
+	"HeartBeatTime": 300000,
+	"HeartBeatMsg": "Core Metadata heart beat",
+	"AppOpenMsg": "This is the EdgeX Core Metadata MicroService",
+	"ConsulHost": "edgex-core-consul",
+	"ConsulCheckAddress": "http://edgex-core-metadata:48081/api/v1/ping",
+	"CheckInterval": "10s",
+	"ConsulPort": "8500",
+	"LoggingFile": "./metadata.log",
+	"LoggingRemoteURL": "http://edgex-core-logging:48061/api/v1/logs",
```
Comment (trcox) [inline res/configuration-docker.json]: edgex-support-logging

File: res/configuration-docker.json [hunk 2]
```diff
@@ -1,33 +1,33 @@
...
+	"NotificationPostDeviceChanges": true,
+	"NotificationsSlug": "device-change-",
+	"NotificationContent": "Device update: ",
+	"NotificationSender": "core-metadata",
```
Comment (trcox) [inline res/configuration-docker.json]: edgex-core-metadata

File: res/configuration-docker.json [hunk 3]
```diff
@@ -1,33 +1,33 @@
...
+	"ApplicationName": "Metadata",
+	"MongoDBName": "metadata",
+	"MongoDBUserName": "meta",
+	"MongoDBPassword": "password",
+	"MongoDBHost": "edgex-mongo",
+	"MongoDBPort": 27017,
+	"MongoDBConnectTimeout": 120000,
+	"ReadMaxLimit": 100,
+	"Protocol": "http",
+	"ServiceName": "core-metadata",
```
Comment (trcox) [inline res/configuration-docker.json]: edgex-core-metadata

File: res/configuration-docker.json [hunk 4]
```diff
@@ -1,33 +1,33 @@
...
-	"NotificationPostDeviceChanges" : true,
-	"NotificationsSlug" : "device-change-",
-	"NotificationContent" : "Device update: ",
-	"NotificationSender" : "edgex-core-metadata",
-	"NotificationDescription" : "Metadata device notice",
-	"NotificationLabel" : "metadata",
-	"supportnotificationsnotificationurl" : "http://edgex-support-notifications:48060/api/v1/notification",
-	"supportnotificationssubscriptionurl" : "http://edgex-support-notifications:48060/api/v1/subscription",
-	"supportnotificationstransmissionurl" : "http://edgex-support-notifications:48060/api/v1/transmission"
+	"ApplicationName": "Metadata",
```
Comment (trcox) [inline res/configuration-docker.json]: edgex-core-metadata

File: res/configuration-docker.json [hunk 5]
```diff
@@ -1,33 +1,33 @@
 {
-	"applicationname" : "edgex-core-metadata",
-	"consulprofilesactive" : "docker;go",
```
Comment (trcox) [inline res/configuration-docker.json:3]: keep this config for consul-client

**Review by trcox — CHANGES_REQUESTED** [review]
See inline comments

**Comment by spencerbull** [comment]
Fixed requested changes by @trcox

**Review by trcox — APPROVED** [review]

**Pull request merged**